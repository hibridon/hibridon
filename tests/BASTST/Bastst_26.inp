     BASTST       
   26                             ibasty
  F  F  F  T                     logdfl, airyfl, readpt, bastst
    1.0    1.0     0.1           rstart, rendld, spac
  F                              prairy
  1.150      3.00    25.0 15.00  tolai, rincr, rendai, fstfac
   1                             nerg
   5000.0
   16.47000                      xmu
   -30.0000                       rcut
  20  20   5   1   0   0   1     jtot1,jtot2,jtotd,jlpar,numin,numax,nud
  48   1                         lscreen, iprint
10  15                        nnout,niout
0  1  2  3  4  5  6  7  8  9
0  1  2  3  4  5  6  7  8  9 10 11 12 13 14 15
  F  F  F  F  F                  prlogd, prsmat, prt2, t2test, wrsmat
  F  F  F  F  F                  wrpart, prpart, prxsec, wrxsec, wavefl
  F  T  F  F  F                  noprin, chlist, ipos, nucros, photof
  T  F  F  F  T                  flaghf, csflag, flagsu, ihomo, twomol
  F  F                           rsflag, boundc
   4   1   2   2                 n1max, j2min, j2min, ipotsy2
 1.4559180 0.35231E-05 -0.0020869 59.3220000  b1rot, d1rot, gamma, b2rot
nofile.dat