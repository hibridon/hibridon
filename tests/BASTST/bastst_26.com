inp=bastst_26.inp
out=bastst_26.out
run
save=save_26.inp