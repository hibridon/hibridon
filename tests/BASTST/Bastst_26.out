      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   = -30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 2SG-1SG system parameters:
 N1MAX  =   4        J2MIN  =   1        J2MAX  =   2        IPOTSY2=   2
 B1ROT  =  1.456     D1ROT  = 3.5231E-06 GAMMA  =-2.0869E-03 B2ROT  =  59.32    
 J1/J2: 
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= T
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= T    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT: 10; J1/J2-OUT: 00  01  02  03  04  05  06  07  08  09  
 INDOUT:    0   1   2   3   4   5   6   7   8   9
           10  11  12  13  14
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 06:45
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: c5993f636e61f7be00d0170107e0d2e6adfb483e
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 06:48:31 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500

   **  CC  2SIGMA-1SIGMA ** RMU= 16.470000  E= 5000.000  JTOT=  20  JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=  -30.00

   N     J1     EPS1    J2     J12      L      EINT(CM-1)
   1     0.5      1      1     0.5     21      118.644
   2     0.5      1      1     1.5     19      118.644
   3     0.5      1      1     1.5     21      118.644
   4     1.5      1      1     0.5     20      121.555
   5     1.5      1      1     1.5     20      121.555
   6     1.5      1      1     1.5     22      121.555
   7     1.5      1      1     2.5     18      121.555
   8     1.5      1      1     2.5     20      121.555
   9     1.5      1      1     2.5     22      121.555
  10     0.5     -1      1     0.5     20      121.558
  11     0.5     -1      1     1.5     20      121.558
  12     0.5     -1      1     1.5     22      121.558
  13     2.5      1      1     1.5     19      127.377
  14     2.5      1      1     1.5     21      127.377
  15     2.5      1      1     2.5     19      127.377
  16     2.5      1      1     2.5     21      127.377
  17     2.5      1      1     2.5     23      127.377
  18     2.5      1      1     3.5     17      127.377
  19     2.5      1      1     3.5     19      127.377
  20     2.5      1      1     3.5     21      127.377
  21     2.5      1      1     3.5     23      127.377
  22     1.5     -1      1     0.5     21      127.383
  23     1.5     -1      1     1.5     19      127.383
  24     1.5     -1      1     1.5     21      127.383
  25     1.5     -1      1     2.5     19      127.383
  26     1.5     -1      1     2.5     21      127.383
  27     1.5     -1      1     2.5     23      127.383
  28     3.5      1      1     2.5     18      136.111
  29     3.5      1      1     2.5     20      136.111
  30     3.5      1      1     2.5     22      136.111
  31     3.5      1      1     3.5     18      136.111
  32     3.5      1      1     3.5     20      136.111
  33     3.5      1      1     3.5     22      136.111
  34     3.5      1      1     3.5     24      136.111
  35     3.5      1      1     4.5     16      136.111
  36     3.5      1      1     4.5     18      136.111
  37     3.5      1      1     4.5     20      136.111
  38     3.5      1      1     4.5     22      136.111
  39     3.5      1      1     4.5     24      136.111
  40     2.5     -1      1     1.5     20      136.119
  41     2.5     -1      1     1.5     22      136.119
  42     2.5     -1      1     2.5     18      136.119
  43     2.5     -1      1     2.5     20      136.119
  44     2.5     -1      1     2.5     22      136.119
  45     2.5     -1      1     3.5     18      136.119
  46     2.5     -1      1     3.5     20      136.119
  47     2.5     -1      1     3.5     22      136.119
  48     2.5     -1      1     3.5     24      136.119
  49     4.5      1      1     3.5     17      147.757
  50     4.5      1      1     3.5     19      147.757
  51     4.5      1      1     3.5     21      147.757
  52     4.5      1      1     3.5     23      147.757
  53     4.5      1      1     4.5     17      147.757
  54     4.5      1      1     4.5     19      147.757
  55     4.5      1      1     4.5     21      147.757
  56     4.5      1      1     4.5     23      147.757
  57     4.5      1      1     4.5     25      147.757
  58     4.5      1      1     5.5     15      147.757
  59     4.5      1      1     5.5     17      147.757
  60     4.5      1      1     5.5     19      147.757
  61     4.5      1      1     5.5     21      147.757
  62     4.5      1      1     5.5     23      147.757
  63     4.5      1      1     5.5     25      147.757
  64     3.5     -1      1     2.5     19      147.766
  65     3.5     -1      1     2.5     21      147.766
  66     3.5     -1      1     2.5     23      147.766
  67     3.5     -1      1     3.5     17      147.766
  68     3.5     -1      1     3.5     19      147.766
  69     3.5     -1      1     3.5     21      147.766
  70     3.5     -1      1     3.5     23      147.766
  71     3.5     -1      1     4.5     17      147.766
  72     3.5     -1      1     4.5     19      147.766
  73     3.5     -1      1     4.5     21      147.766
  74     3.5     -1      1     4.5     23      147.766
  75     3.5     -1      1     4.5     25      147.766
ILAM=   1 LAMNUM(ILAM) =       0
ILAM=   2 LAMNUM(ILAM) =       0
ILAM=   3 LAMNUM(ILAM) =       0
 *** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS       0
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    1.4 %
      CURRENT DATE:  Mar 19 06:48:31 2023
 ===============================================================================
